packages/scaffolding-cli/src/__tests__/domain/config/worker_maps/ssr_aks_tfs.test.ts

```diff
@@ -3,31 +3,36 @@ import { BusinessSection, CloudSection } from "../../../../domain/model/prompt_a
 import { ssr } from '../../../../domain/config/worker_maps'
 import conf from  '../../../../domain/config/static.config.json'
 import { Static, FolderMap } from '../../../../domain/model/config';
-let staticConf: Static = conf as Static;
+
+const staticConf: Static = conf as Static;
         
-let projName = "test-app-1"
-let biz: BusinessSection = <BusinessSection>{
+const projName = "test-app-1"
+
+const biz = {
     company: "test",
     project: "test2",
     component: "test3",
     domain: "domain"
-}
+} as BusinessSection
 
-let cloud: CloudSection = <CloudSection>{
+const cloud = {
     region: "uksouth",
     resourceGroup: "my-rg"
-}
+} as CloudSection
 
-let files: Array<BuildReplaceInput> = [
+const files: Array<BuildReplaceInput> = [
     {
         files: ["**/package.json"],
         values: {
-            "PROJECT_NAME": projName
+            "PROJECT_NAME": projName,
```

Comment:
Duplication? Make it case insensitive?

Reply:
yeah that would just increase the replace cycle time, rather would replace all occurences in caps into lower case for the node app case